molo/profiles/migrations/0004_add_profile_settings.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-06-30 12:14
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0003_add_email_field'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='userprofilessettings',
+            name='prevent_email_in_username',
+            field=models.BooleanField(default=False, verbose_name='Prevent email in username / display name'),
+        ),
+        migrations.AddField(
+            model_name='userprofilessettings',
+            name='prevent_number_in_username',
```

Comment:
I would call this `prevent_phone_number_in_username` and change the `verbose_name` too so that it's clear what kind of number we're talking about here, ie: not just any number.